flocker/control/_model.py

```diff
@@ -314,6 +314,10 @@ class Application(PClass):
     :ivar command_line: Custom command to run using the image, a ``PVector``
         of ``unicode``. ``None`` means use default.
 
+    :ivar swappiness: Tunable swappiness of the container (default is None,
+        which disables swapping). Valid value is in between (including)
+        [0, 100].
+
```

Comment:
:question: Maybe check the default behaviour. Docker docs say:
> By default, if you are not using --memory-swappiness, memory swappiness value will be inherited from the parent.

* https://docs.docker.com/engine/reference/run/#/swappiness-constraint